x-pack/plugins/enterprise_search/server/routes/workplace_search/groups.test.ts

```diff
@@ -392,4 +393,11 @@ describe('groups routes', () => {
       });
     });
   });
+
+  describe('registerGroupsRoutes', () => {
+    it('runs without errors', () => {
+      const mockRouter = new MockRouter({} as any);
+      registerGroupsRoutes({ ...mockDependencies, router: mockRouter.router });
+    });
+  });
```

Comment:
Are you sure you need this? `registerGroupsRoutes` should already be covered by index.test..ts

Reply:
Nice catch! Fixed